proposals/0091-SDLScreen-SDLWindow-Projection.md

```diff
@@ -1,110 +1,209 @@
+SDLCarWindow uses the existing SDLStreamingMediaManager created by SDLLifecycleManager to manage video streaming.
+SDLStreamingMediaManager is used to project the contents of the virtual external display. 
 
-```objc
-@interface UIButton (SDLFocusable)
+**Touch Events**
 
-@property(nonatomic, readonly) BOOL canBecomeFocused;
+Ownership of SDLTouchManager moves from SDLStreamingMediaLifecycleManager to SDLCarWindow. SDLStreamingMediaLifecycleManager links to SDLCarWindow to retain access to SDLTouchManager. 
 
-@end
-```
+SDLTouchManager handles each touch event by first giving SDLTouchManagerDelegate delegate a chance to handle it. If the event remains unhandled, SDLTouchManager will attempt to translate the event directly into a view action and invoke that action. By translating touch events into view actions within SDLTouchManager, no additional effort is required on the part of the app developer to handle touch events. This is provided for UIKit view types only. 
 
-For head units that allow the app to manage focus and selection, the SDLInterfaceManager provides the local logic to do so and interacts with the views through the [UIFocusEnvironment protocol](https://developer.apple.com/documentation/uikit/uifocusenvironment).
- 
-The SDLCarWindow class will also include the previously app-level common logic that interacts with the existing SDLStreamingMediaManager class to stream pixel buffers and receive audio data. It relies on the helper class SDLStreamingMediaLifecycleManager to ensure streaming is allowed and manage the streaming state.
+Currently supported types are: UIButton, UIControl, UINavigationViewController, UITableView, UICollectionViewCell, UIScrollView, UISearchBar, UITextField, UISegmentedControl and UITabBarController. [2]
 
-```objc
-extern NSString *const SDLVideoStreamDidStartNotification;
-extern NSString *const SDLVideoStreamDidStopNotification;
+Touch events for custom views defined by the app must be handled by the app. An app wishing to handle touch events must adopt and implement the SDLTouchManagerDelegate protocol. The app will then have the choice of handling a touch event, or passing the event on to SDL to handle it automatically. When the app handles the event, it returns YES from the delegate method. A return value of NO instructs SDL to handle the event.
 
-extern NSString *const SDLAudioStreamDidStartNotification;
-extern NSString *const SDLAudioStreamDidStopNotification;
+In a later proposal, SDL will be extended to include focus and selection control on the handset, following the UIFocusEngine model from tvOS. [3]
```

Comment:
[3] does not link to the note at the end of the proposal (not sure if this is intentional).

Reply:
Removed notes